src/calliope/backend/__init__.py

```diff
@@ -1 +1,33 @@
-"""Callipe's backend module."""
+"""Calliope's optimisation backend module."""
+
+import xarray as xr
+
+from calliope.backend.latex_backend_model import MathDocumentation
+from calliope.backend.parsing import ParsedBackendComponent
+from calliope.backend.pyomo_backend_model import PyomoBackendModel
+from calliope.exceptions import BackendError
+
+MODEL_BACKENDS = ("pyomo",)
+
+
+def get_model_backend(name: str, data: xr.Dataset, **kwargs):
```

Comment:
This should have a return type hint

Reply:
Unfortunately this is where the nasty `TYPE_CHECKING` issue occurs.
I have not found a good way of avoiding it...

I did add it to the docstring, but the only way I've found to fix it would involve changing the circular typing issue between helper functions and the generic backend file.

Reply:
Hmm, and `IF TYPE_CHECKING` and then stringifying the type hint won't do it?

Reply:
Oh, that is a neat trick! I thought it would only work for `type["XXXXX"]` cases, for some reason.